packages/tasks/src/model-libraries.ts

```diff
@@ -429,6 +429,12 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		filter: false,
 		countDownloads: `path:"tokenizer.model"`,
 	},
+	saelens: {
+		prettyLabel: "SAELens",
+		repoName: "SAELens",
+		repoUrl: "https://github.com/jbloomAus/SAELens",
+		filter: false,
```

Comment:
QQ: @Wauplin - do you have any recommendations on the best way to track downloads in this scenario: https://huggingface.co/jbloom/GPT2-Small-SAEs-Reformatted/tree/main

Reply:
I'm afraid it is a bit more complicated, on more newer repos (cannot link them atm as they are not public yet) they might just have an `npz` file. The usage is more layer wise so I think the former suggestion for `npz`, `cfg`, `json` makes more sense.

Reply:
Yes, I think the each block provides a different lens so they are used independently. But the structure does not appear to be standardized, so I would merge as is for now (without download counts) and clarify when we can.

Reply:
okay, the downloads would need more time to standardise. Given this unblocks a release happening soon. Is it okay if we merge as is for now and revisit in a day when we have more clarity.

Reply:
Fine for me!